views/basket/basket.erb

```diff
@@ -0,0 +1,36 @@
+<h1>Your basket</h1>
+<table class="table table-hover table-condensed">
+  <thead>
+    <tr>
+      <th>ID</th>
+      <th>Name</th>
+      <th>Quantity</th>
+      <th>Price/ea(netto)</th>
+      <th>Total netto</th>
+      <th>Total brutto</th>
+    </tr>
+  </thead>
+  <tbody>
+    <% @basket_products.each do |pr| %>
+    <tr>
+      <td><%= pr.id %></td>
+      <td><a href="/<%= pr.id %>"><%= pr.name %></a></td>
+      <td><%= pr.quantity %></td>
+      <td><%= pr.price %></td>
+      <td><%= pr.netto_price.round(2) %></td>
+      <td><%= pr.brutto_price.round(2) %></td>
+      <td><a href="/<%= pr.id %>/delete">Delete</a></td>
```

Comment:
delete should not be a link (GET call) but form that will send DELETE method on given address